modules/quotas-creating-object-count-quotas.adoc

```diff
@@ -24,9 +21,7 @@ To configure an object count quota for a resource:
 $ oc create quota <name> \
     --hard=count/<resource>.<group>=<quota>,count/<resource>.<group>=<quota> <1>
 ----
-<1> `<resource>` is the name of the resource, and `<group>` is the API group, if
-applicable. Use the `oc api-resources` command for a list of resources and
-their associated API groups.
+<1> `<resource>` is the name of the resource, and `<group>` is the API group, if applicable. Use the `oc api-resources` command for a list of resources and their associated API groups.
```

Comment:
ISG suggests rewriting so that a sentence doesn't start with lowercase. So maybe something like this?

> <1> The `<resource>` variable is the name of the resource, and `<group>` is the API group, if applicable. Use the `oc api-resources` command for a list of resources and their associated API groups.